src/main/java/com/example/eatmate/app/domain/notice/domain/Notice.java

```diff
@@ -0,0 +1,47 @@
+package com.example.eatmate.app.domain.notice.domain;
+
+import com.example.eatmate.app.domain.notice.dto.NoticeAdminRequestDto;
+import com.example.eatmate.global.common.BaseTimeEntity;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import lombok.AccessLevel;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+@Entity
+@Getter
+public class Notice extends BaseTimeEntity {
+
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	private Long id;
+
+	private String title;
+
+	@Column(columnDefinition = "TEXT")
+	private String content;
+
+	@Builder
+	private Notice(NoticeAdminRequestDto noticeAdminRequestDto) {
+		this.title = noticeAdminRequestDto.getTitle();
+		this.content = noticeAdminRequestDto.getContent();
+	}
+
+	public static Notice createNotice(NoticeAdminRequestDto noticeAdminRequestDto) {
+		return Notice.builder()
+			.noticeAdminRequestDto(noticeAdminRequestDto)
+			.build();
+	}
+
+	public void update(NoticeAdminRequestDto noticeAdminRequestDto) {
+		this.title = noticeAdminRequestDto.getTitle();
+		this.content = noticeAdminRequestDto.getContent();
+	}
+
+}
```

Comment:
도메인 계층의 엔티티가 DTO를 직접 참조하고 있는데, 이는 계층 간 의존성을 발생시키고 단일 책임 원칙(SRP)에도 위배된다고 생각됩니다!

```suggestion
	@Builder
	private Notice(String title, String Content) {
		this.title = title;
		this.content = content;
	}

	public static Notice createNotice(String title, String content) {
		return Notice.builder()
			.title(title)
			.content(content)
			.build();
	}

	public void update(String title, String content) {
		this.title = title;
		this.content = content;
	}

}
```

Reply:
아, DTO를 직접 참조하면 더 간단해질 수 있다고 생각했는데 그런 문제가 있군요! 수정하겠습니다